### PR TITLE
DUPLO-19625 TF Doc:duplocloud_asg_profile: Add allowed metrics values for `enabled_metrics` parameter and add example in example usage.

### PR DESCRIPTION
**Pull request by Duplo-nikv**
## Overview

Updated document for duplocloud_asg_profile resource.
## Summary of changes

This PR does the following:

- Added enable metrics example
- Updated Description by listing allowed metrics

## Testing performed

- [ ] Using unit tests
- [ ] Manually, on my local system
- [ ✓] Manually, on a remote test system

## Describe any breaking changes

- ...

___

### **PR Type**
Documentation

___

### **Description**
- Updated the `enabled_metrics` parameter description in the ASG profile schema to include a comprehensive list of allowed metrics.
- Added an example in the documentation to demonstrate how to enable metrics for an ASG.
- Provided a new example in the Terraform configuration file for enabling specific metrics in an ASG.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>resource_duplo_asg_profile.go</strong><dd><code>Update ASG profile schema with allowed metrics list</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

duplocloud/resource_duplo_asg_profile.go

<li>Updated the description for the <code>enabled_metrics</code> parameter.<br> <li> Listed all possible metrics that can be enabled.<br>

</details>

  </td>
  <td><a href="https://github.com/duplocloud/terraform-provider-duplocloud/pull/635/files#diff-f58c7134634373bfbd4607fa72e0d5fb28a0da60d136394bacd5671ea75c8346">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>asg_profile.md</strong><dd><code>Add example and update description for ASG metrics</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/resources/asg_profile.md

<li>Added an example usage for enabling metrics in ASG.<br> <li> Updated the description for <code>enabled_metrics</code> with allowed values.<br>

</details>

  </td>
  <td><a href="https://github.com/duplocloud/terraform-provider-duplocloud/pull/635/files#diff-0b0507a0a18d1275d81b42d11e06ba436e7ef85ec5c0cb56cc83b8e6381c2909">+25/-1</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>resource.tf</strong><dd><code>Add example for enabling ASG metrics</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/resources/duplocloud_asg_profile/resource.tf

- Added a new example for enabling metrics in ASG.

</details>

  </td>
  <td><a href="https://github.com/duplocloud/terraform-provider-duplocloud/pull/635/files#diff-7ace589fb0c38f4be1cd00ae3d546f42be9c5c00436f10ba695e6495e8645f42">+24/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: docs/resources/asg_profile.md [hunk 1]
```diff
@@ -57,6 +81,27 @@ resource "duplocloud_asg_profile" "duplo-test-asg" {
 - `can_scale_from_zero` (Boolean) Whether or not ASG should leverage duplocloud's scale from 0 feature
 - `cloud` (Number) The numeric ID of the cloud provider to launch the host in. Defaults to `0`.
 - `enabled_metrics` (List of String) List of metrics to collect for the ASG
```
Comment (tahir-duplo) [inline docs/resources/asg_profile.md:83]: We can just add comm separated inside back tick like `GroupMinSize, GroupMaxSize`

File: duplocloud/resource_duplo_asg_profile.go [hunk 2]
```diff
@@ -92,9 +92,31 @@ func autoscalingGroupSchema() map[string]*schema.Schema {
 	}
 
 	awsASGSchema["enabled_metrics"] = &schema.Schema{
```
Comment (tahir-duplo) [inline duplocloud/resource_duplo_asg_profile.go:94]: Add validation for this attribute

File: docs/resources/asg_profile.md (hunk 1 above)
Comment (Duplo-nikv) [reply docs/resources/asg_profile.md]: done

File: duplocloud/resource_duplo_asg_profile.go (hunk 2 above)
Comment (Duplo-nikv) [reply duplocloud/resource_duplo_asg_profile.go:94]: done

**Pull request merged**